### PR TITLE
Examples: Add python examples

### PR DESCRIPTION
**Pull request by dp834**

### CODE REVIEW

**Review by kumsu-bear — APPROVED** [review]

File: examples/python/manual_drive.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import argparse
```
Comment (aqchin) [inline examples/python/manual_drive.py:1]: File name should be `drive_robot.py` and the BUILD file

File: examples/python/manual_drive.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+import grpc
+import time
+
+from bearrobotics.api.v0.robot import robot_api_service_pb2_grpc
+from bearrobotics.api.v0.robot import robot_api_service_pb2
+from bearrobotics.api.v0.common import math_pb2
```
Comment (aqchin) [inline examples/python/manual_drive.py:7]:
```suggestion
import argparse
import time

import grpc

from bearrobotics.api.v0.common import math_pb2
from bearrobotics.api.v0.robot import robot_api_service_pb2
from bearrobotics.api.v0.robot import robot_api_service_pb2_grpc
```

File: examples/python/get_system_info.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import grpc
+from bearrobotics.api.v0.robot import robot_api_service_pb2_grpc
+from bearrobotics.api.v0.robot import robot_api_service_pb2
```
Comment (aqchin) [inline examples/python/get_system_info.py:3]:
```suggestion
import grpc

from bearrobotics.api.v0.robot import robot_api_service_pb2
from bearrobotics.api.v0.robot import robot_api_service_pb2_grpc
```

**Review by aqchin — APPROVED** [review]

**Comment by CaptainLee-Bear** [comment]
A fly-by comment. A nice job adding examples! Please update the PR title to include a Jira tag or a project id instead of NOJIRA e.g. `[API]`:

Reference: 
https://bearrobotics.slack.com/archives/C047YUU362K/p1733262684977089?thread_ts=1733214471.995159&cid=C047YUU362K

**Review by aqchin — APPROVED** [review]

**Pull request merged**